### DISCUSSION TITLE
[iOS] Pod install error Unable to find a specification for `React-callinvoker` depended upon by `RNReanimated`

### DISCUSSION THREAD

**Discussion by ihiteshkumar** [software-mansion/react-native-reanimated#2397] [Q&A]
Getting this error when pod install on iOS

Unable to find a specification for `React-callinvoker` depended upon by `RNReanimated`

React-callinvoker is already available in pod:

    pod 'ReactCommon/callinvoker', :path => "../../common/ReactNative/libs/react-native/ReactCommon"

![Screenshot 2021-09-09 at 1 59 13 PM](https://user-images.githubusercontent.com/9604287/132651664-897aa3e9-b598-45d3-b3cc-8aaac2373477.png)

**Comment by abdulkarimkhaled** [comment]
Is there any update on this ?

**Reply by mehavel** [reply]
where you able to find a solution. I am struck at the same place
